src/main/scala/io/iohk/ethereum/vm/PrecompiledContracts.scala

```diff
@@ -0,0 +1,101 @@
+package io.iohk.ethereum.vm
+
+import akka.util.ByteString
+import io.iohk.ethereum.crypto._
+import io.iohk.ethereum.domain.Address
+import GasFee.wordsForBytes
+import io.iohk.ethereum.utils.ByteUtils
+
+import scala.util.Try
+
+// scalastyle:off magic.number
+object PrecompiledContracts {
+
+  val EcDsaRecAddr = Address(1)
+  val Sha256Addr = Address(2)
+  val Rip160Addr = Address(3)
+  val IdAddr = Address(4)
```

Comment:
It's a number of a fixed bit size, so yeah, just like UInt256 its byte repr includes the zeroes.